database/migrations/2024_04_24_165732_create_languages_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('languages', function (Blueprint $table) {
+            $table->uuid('id')->primary();
+            $table->string('language_name')->nullable();
+            $table->enum('language_level', ['Bàsic', 'Intermedi', 'Avançat', 'Natiu'])->nullable();
+            $table->unique(['language_name', 'language_level']);
```

Comment:
I would not have thought of it. :boom:  :+1:

Reply:
Yeahh! I'm also surprised! But really not, this comes from @toniGitH...